envoy/admin/admin.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
```

Comment:
This should be `envoy/admin/v2/admin.proto` as per versioning elsewhere in data-plane-api.

Reply:
Kk. So basically the whole repo is v2?

Reply:
At this point, yep. In the future we may have independent subtrees evolve at different rates, but this is probably medium-long term.